acl-lib/test/cmr/acl/test/core_test.clj

```diff
@@ -43,42 +44,46 @@
             (is (= (fun headers) expected-client-id)))
 
           "user agent is mozilla"
-          {"user-agent" "Mozilla."} acl_core/BROWSER_CLIENT_ID
+          {"user-agent" "Mozilla."} acl-core/BROWSER_CLIENT_ID
 
           "user agent is curl"
-          {"user-agent" "curl."} acl_core/CURL_CLIENT_ID
+          {"user-agent" "curl."} acl-core/CURL_CLIENT_ID
 
           "user agent is opera"
-          {"user-agent" "Opera."} acl_core/BROWSER_CLIENT_ID
+          {"user-agent" "Opera."} acl-core/BROWSER_CLIENT_ID
 
           "client id does not exist"
-          {} acl_core/UNKNOWN_CLIENT_ID
+          {} acl-core/UNKNOWN_CLIENT_ID
 
           "unknown client id"
-          {"user-agent" "Other"} acl_core/UNKNOWN_CLIENT_ID)))
+          {"user-agent" "Other"} acl-core/UNKNOWN_CLIENT_ID)))
 
 (deftest add-authentication-to-context-test
   (let [context {}
         params {:token "params-token"}
         headers {"user-agent" "curl."}
         fun #'cmr.acl.core/add-authentication-to-context
-        expected-context {:token "params-token" :client-id acl_core/CURL_CLIENT_ID}]
+        expected-context {:token "params-token" :client-id acl-core/CURL_CLIENT_ID}]
     (println (fun context params headers))
     (is (= (fun context params headers) expected-context))))
 
 (deftest context->sids-test
   (testing "get sids from requesting it"
     (let [context {:keys ["params-token"]}]
-    (with-redefs [cmr.acl.core/request-sids (fn [context] [:guest])]
-      (is (= (acl_core/context->sids context) [:guest]))))
+    (with-redefs [cmr.acl.core/request-sids (fn [_context] [:guest])]
+      (is (= (acl-core/context->sids context) [:guest]))))
   (testing "get sids from context"
     (let [context {:sids [:guest]}]
-    (is (= (acl_core/context->sids context) [:guest]))))))
+    (is (= (acl-core/context->sids context) [:guest]))))))
 
 (deftest get-permitting-acls-test
   (testing "Exception getting acls"
-    (with-redefs [cmr.transmit.access-control/acl-type->acl-key (fn [object-identity-type] (throw (Exception. "Exception to break test")))]
-      (is (nil? (acl_core/get-permitting-acls nil nil nil nil)))))
+    (with-redefs [cmr.transmit.access-control/acl-type->acl-key 
```

Comment:
cmr.transmit.access-control :as access-control was added to require block so let's make the transition here away from the fully qualified namespace